packages/experimental/TabList/src/TabListAnimatedIndicator/useAnimatedIndicatorStyles.ts

```diff
@@ -45,5 +45,7 @@ export function useAnimatedIndicatorStyles(): AnimatedIndicatorStyles {
     };
   }, [vertical, selectedIndicatorLayout, animatedIndicatorStyles]);
 
+  React.useEffect(() => setIndicatorHasRendered(styles !== null), [setIndicatorHasRendered, styles]);
```

Comment:
Is there a comment somewhere that explains that there's a static indicator rendered to cover the late render of the animated indicator? If not it would be great to add a comment to make the intent of this variable clear.

Reply:
There's a comment in Tab.styling.tsx, but I can add one here as well.

Reply:
Ah ok. I would say it would be nice to add more about the intent of the static indicator, i.e. it exists to cover the animated indicator "popping in," rather than just that it'll be swapped out once styles are generated.

Reply:
New comment looks great, thanks!